govc/vm/disk/create.go

```diff
@@ -113,10 +115,31 @@ func (cmd *create) Run(ctx context.Context, f *flag.FlagSet) error {
 		return nil
 	}
 
+	backing := disk.Backing.(*types.VirtualDiskFlatVer2BackingInfo)
+
 	if cmd.Thick {
-		backing := disk.Backing.(*types.VirtualDiskFlatVer2BackingInfo)
 		backing.ThinProvisioned = types.NewBool(false)
-		backing.EagerlyScrub = types.NewBool(cmd.Eager)
+	}
+
+	backing.EagerlyScrub = types.NewBool(cmd.Eager)
+
+	switch cmd.DiskMode {
```

Comment:
"dependent" isn't listed in the docs: http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.wssdk.apiref.doc/vim.vm.device.VirtualDiskOption.DiskMode.html

Should be ok just to leave it out of the usage above.  Then you can also use the same []string{...} for usage and validation.